launcher_go/v2/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/alecthomas/kong"
+	"github.com/discourse/discourse_docker/launcher_go/v2/utils"
+	"golang.org/x/sys/unix"
+	"os"
+	"os/exec"
+	"os/signal"
+)
+
+type Cli struct {
+	Version      kong.VersionFlag `help:"Show version."`
+	ConfDir      string           `default:"./containers" help:"Discourse pups config directory." predictor:"dir"`
+	TemplatesDir string           `default:"." help:"Home project directory containing a templates/ directory which in turn contains pups yaml templates." predictor:"dir"`
+	BuildDir     string           `default:"./tmp" help:"Temporary build folder for building images." predictor:"dir"`
+	ForceMkdir   bool             `short:"p" name:"parent-dirs" help:"Create intermediate output directories as required.  If this option is not specified, the full path prefix of each operand must already exist."`
```

Comment:
I'm wondering if there is a need to make this configurable. Is it possible for us to just use `/tmp` instead as I feel like these two options can be quite confusing to users.

Reply:
Perhaps we can just make it such that any parent directories we need in `./tmp` is created if it isn't present? I'm not sure if there is value in making this configurable.

Reply:
I've hidden the build dir option and removed the force option (defaults to making parent dirs as needed)